CMS-Enterprise/easi-app
# [EASI-4489] System intake document upload form

# EASI-4489

[Figma](https://www.figma.com/design/G0UN35V7SY0g9nkTIESxsY/EASi-GRB-Experience?node-id=2827-9638&t=5toFxvXRMjFCr9ph-0)

## Description
- Updated system intake document upload form to match Figma
  - Added `type` prop to component to differentiate between admin and requester forms
  - GRB email notification question will be added in separate ticket
- New route for admin document upload form (/governance-review-team/:systemId/documents/upload)


## How to test this change
- Test both admin and requester document upload forms
- Requester form should return to documents form step on submit
- Admin form should return to GRB Review tab on submit

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
